tensorflow_recommenders/layers/factorized_top_k.py

```diff
@@ -687,25 +685,18 @@ def index(
     self._serialized_searcher = self._build_searcher(
         candidates).serialize_to_module()
 
```

Comment:
Please remove one of these empty lines, so that there is only one left.

Reply:
thanks!